nova/virt/libvirt/driver.py

```diff
@@ -8677,8 +8693,7 @@ def _has_numa_support(self):
 
         if (caps.host.cpu.arch in (fields.Architecture.I686,
                                    fields.Architecture.X86_64,
-                                   fields.Architecture.AARCH64) and
-                self._host.has_min_version(hv_type=host.HV_DRIVER_QEMU)):
```

Comment:
I think, it would be better to test here for both `HV_DRIVER_QEMU` and `HV_DRIVER_CH`.
LXC and Xen do not support numa architecutres

Reply:
There currently isn't a `HV_DRIVER_CH` variable defined. for qemu there is `HV_DRIVER_QEMU = "QEMU"`.

Following check does the comparison later on, where `hv_type ` is set to `HV_DRIVER_QEMU`:
```python
            if hv_type is not None:
                hypervisor_type = conn.getType()
                if hypervisor_type != hv_type:
                    return False
```

Do you have the possibility to check what `conn.getType()` returns in the case CHV is in use?